src/archiefvernietigingscomponent/templates/master.html

```diff
@@ -14,6 +14,9 @@
         <link href="{% static 'css/screen.css' %}" media="screen" rel="stylesheet" type="text/css" />
         <link href="{% static 'hijack/hijack-styles.css' %}" rel="stylesheet" type="text/css" />
         {% block extra_css %}{% endblock %}
+        <style type="text/css">
+            {% theme_stylesheet %}
```

Comment:
The `theme_stylesheet` already contains <style .. > tag. So maybe one of the 2 should be removed?